src/components/ec-option-card/ec-option-card.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <component
+    :is="componentTag"
+    v-bind="{
+      ...attrs,
+      ...componentProps,
+      'data-test': attrs['data-test'] ? `${attrs['data-test']} ec-option-card` : 'ec-option-card',
+    }"
+    class="ec-option-card"
+    :class="getOptionCardClasses()"
+    @click="setClickAction()"
+  >
+    <span
+      data-test="ec-option-card__title"
+    >
+      <ec-icon
+        v-if="optionCardIconName && optionTitle"
```

Comment:
why we check `optionTitle` here?

Reply:
Because the icon is ever only visible with a title and we can have a title without an icon. This check prevents people from pass an icon but not a title, this would avoid that scenario.